01_server/db/models/user.js

```diff
@@ -1,28 +1,28 @@
 'use strict';
 var crypto = require('crypto');
 var mongoose = require('mongoose');
+var Sighting = mongoose.model('Sighting');
 
-var schema = new mongoose.Schema({
+var userSchema = new mongoose.Schema({
+
+    name: { 
+        type: String
+    },
     email: {
         type: String
     },
     password: {
         type: String
     },
+    sighting: {
+        type: mongoose.Schema.Types.ObjectId,
```

Comment:
the only change is that this should be an array of sightings type: [mongoose.Schema.Types.ObjectId]